src/list/ListItem.js

```diff
@@ -213,6 +241,12 @@ styles = StyleSheet.create({
     alignItems: 'flex-end',
     justifyContent: 'center'
   },
+  switchContainer: {
+    flex: 0.15,
```

Comment:
@Joshuabaker2 All good. Once the merge conflict is fixed. I will merge it. Thanks again